src/lib/markdown.ts

```diff
@@ -118,6 +120,21 @@ const anchorMarkedExtension = {
   },
 };
 
+export const walkTokens = (token: marked.Tokens.Generic) => {
+  if (token.type !== "code" && token.type !== "codespan") {
+    if (
+      "text" in token &&
+      !token.text.match(new RegExp(`${trustedHtmlTags.join("|")}`, "g"))
```

Comment:
What exactly is `text`? Is it `<foo>` and `</foo>`, for example? If so, why not just replace the contents in one go, instead of matching first? Also, I think closing tags are not handled by this code?

Reply:
> What exactly is text? 
`token.text` is what the markdown tokenizer categorizes as `text` and it depends on each token, here you can see some definitions and which one have and which one don't have the text property.
https://github.com/markedjs/marked/blob/450da06c7c8466958002e5c1744c91d02b300e7a/src/Tokenizer.js#L67
In our case e.g. we don't want to match against `<hr/`> or a whitespace, that's why I check for `token.text`

>  If so, why not just replace the contents in one go, instead of matching first.
I matched against a regex to skip the tokens we don't want to escape, but we could probably also avoid regex and check like `!trustedHtmlTags.includes(token.text)`.

>  closing tags are not handled by this code
Closing tags are handled since we match against the inner text and don't include a closing slash e.g. `<code> === </code>` for us 😁

Reply:
Ok just tried the 

> I matched against a regex to skip the tokens we don't want to escape, but we could probably also avoid regex and check like !trustedHtmlTags.includes(token.text).

And I think it's easier to match the token out of the html tag and handle the closing tags instead of trying to match with string or array methods..

Reply:
Okay found this solution that I think is nice and we can avoid another regex expression and matching
https://github.com/radicle-dev/radicle-interface/pull/846/commits/50c24f592963ef98508d0245b6682acc70b9e4a4